src/api/vdom.js

```diff
@@ -44,17 +44,22 @@ function applyEvent (elem, ename, newFunc) {
     events = elem[$currentEventHandlers] = {};
   }
 
-  const oldFunc = events[ename];
-
-  // Remove old listener so they don't double up.
-  if (oldFunc) {
-    elem.removeEventListener(ename, oldFunc);
+  // Undefined indicates that there is no listener yet.
+  if (typeof events[ename] === 'undefined') {
```

Comment:
No, if it's `undefined`, that means there's no listener yet, so we must bind one in order to trigger anything that gets bound via a property. Later we set the handler in the `events` map.

This is a workaround for https://github.com/webcomponents/shadydom/issues/24 but I like the logic much better anyways. There'd be pretty much zero overhead for adding a single noop listener, and optimising for the common use case, generally (from experience) once you add a listener, you're going to keep it there.